src/common/droid-util.c

```diff
@@ -2411,21 +2433,22 @@ pa_droid_stream *pa_droid_stream_ref(pa_droid_stream *s) {
 
 void pa_droid_stream_unref(pa_droid_stream *s) {
     pa_assert(s);
+    pa_assert(s->input || s->output);
     pa_assert(PA_REFCNT_VALUE(s) >= 1);
 
     if (PA_REFCNT_DEC(s) > 0)
         return;
 
-    if (s->out) {
+    if (s->output) {
         pa_mutex_lock(s->module->output_mutex);
         pa_idxset_remove_by_data(s->module->outputs, s, NULL);
-        s->module->device->close_output_stream(s->module->device, s->out);
+        s->module->device->close_output_stream(s->module->device, s->output->stream);
```

Comment:
consistency: sometimes stream pointer is cleared after close, sometimes not - does it matter?